### PR TITLE
Frontend/route cleanup

### PR DESCRIPTION
**Pull request by herrickfeng**
just some route cleanup on the apps,js page
 
will still have to go through them again once everyones finished to convert them to private routes

### CODE REVIEW

File: backend/.env.example [hunk 1]
```diff
@@ -1,11 +0,0 @@
-# Stage to run the application as [dev|test|prod]
```
Comment (mitcmurph) [inline backend/.env.example:1]: Dont delete this please

**Review by mitcmurph — CHANGES_REQUESTED** [review]

File: frontend/src/App.js [hunk 2]
```diff
@@ -48,97 +59,87 @@ function AppProvider(props) {
...
         exact={true}
-        component={ViewRecipe}>
-      </Route>
+        component={Dashboard} />
+      <PrivateRoute path="/dashboard/create-course"
+        exact={true}
+        component={CreateNewCourse} />
+      <PrivateRoute path="/dashboard/view-training"
+        exact={true}
+        component={ViewTrainingPage} />
```
Comment (mitcmurph) [inline frontend/src/App.js:103]: Can you address everything in the PR you closed #87

**Review by mitcmurph — CHANGES_REQUESTED** [review]

File: backend/.env.example (hunk 1 above)
Comment (herrickfeng) [reply backend/.env.example:1]: didnt touch the .env example not sure what to change

File: frontend/src/App.js (hunk 2 above)
Comment (herrickfeng) [reply frontend/src/App.js:103]: those changes are being done in the that branch will push as soon as its done

File: backend/.env.example (hunk 1 above)
Comment (mitcmurph) [reply backend/.env.example:1]: you deleted it in your first commit view training draft lol

Comment (mitcmurph) [reply backend/.env.example]: I have undeleted it

File: frontend/src/App.js (hunk 2 above)
Comment (mitcmurph) [reply frontend/src/App.js:103]: OK next time branch off master please, not uncompleted changes.

**Review by mitcmurph — APPROVED** [review]

**Comment by mitcmurph** [comment]
This is good, I think we should group the pages into folders in a similar way.

**Comment by herrickfeng** [comment]
Those changes are being made in the view training branch

**Pull request merged**